### ISSUE TITLE
Could not open env for DB (-30793) (error code -30793)

### ISSUE THREAD

**Issue by Lissten** [objectbox/objectbox-java#761]
```
io.objectbox.exception.DbException: Could not open env for DB (-30793) (error code -30793)
at io.objectbox.BoxStore.nativeCreate(Native Method)
at io.objectbox.BoxStore.(init)(BoxStore.java:195)
at io.objectbox.BoxStoreBuilder.build(BoxStoreBuilder.java:352)
at com.eebbk.comicbookread.base.ComicBookReadApplication.initObjectBox(ComicBookReadApplication.java:89)
at com.eebbk.comicbookread.base.ComicBookReadApplication.lambda$new$0(ComicBookReadApplication.java:198)
at com.eebbk.comicbookread.base.-$$Lambda$ComicBookReadApplication$u_dnV7ijwAaWt2xFhjtd6eq8RAE.run(lambda)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
at java.lang.Thread.run(Thread.java:761)
```

**Comment by greenrobot-team** [comment]
Check #370 and #425 for potential solutions. It looks like you might be initializing ObjectBox multiple times (on each thread).

If that won't help, we need more information about what you are doing and what your project setup looks like.

**Comment by Lissten** [comment]
I'm initializing in a child thread, is that relevant?

**Comment by greenrobot-team** [comment]
If it's done only once for the lifetime of the app (or until the `BoxStore` is closed again), then no.

**Comment by Hatzen** [comment]
I got the same error code while exporting and importing data. In my case i forget to close the boxstore which was opened within global scope (android application) and still open while manipulating files. After closing via `boxStore.close()` everything worked fine.   
To be 100% sure boxStore is closed i ran following methods, also just using boxStore.close() will do the trick.. (The following lines are kotlin syntax)
``` java
    private fun forceCloseBoxStore(boxStore: BoxStore) {
        boxStore.closeThreadResources()
        boxStore.close()
        while(!boxStore.isClosed);
    }
```

**Comment by greenrobot-team** [comment]
Closing this issue due to inactivity. :zzz: Feel free to re-open with more details or submit a new issue.
